frontend/src/model/Cocktail.ts

```diff
@@ -0,0 +1,5 @@
+export type Cocktail = {
+    strDrink : string, //name
+    strDrinkThumb : string, //image
```

Comment:
```suggestion
    strDrinkThumb : string
```
wofür steht das str vor der Bezeichnung? für String? Würde ich dann lieber weglassen. Bin mir auch nicht ganz sicher, ob ich wüsste was DrinkThumb ist, könnte die Bezeichnung etwas eindeutiger sein?